workflow/tools/util_in_docker/generate_song_payload.py

```diff
@@ -25,6 +25,11 @@ def main():
     parser.add_argument('lane_unaligned_dir')
     parser.add_argument('oxog_metrics_dir')
     parser.add_argument('multiple_metrics_dir')
+    parser.add_argument('--wf:name', dest="wf_name", required=True)
```

Comment:
having `:` in the argument name seems uncommon, can we just using hyphen? like: `--wf-name` instead of `--wf:name`. This way, we don't even need to define `dest`